test/zippy/mzcompose.py

```diff
@@ -115,11 +119,20 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         help="Cockroach DockerHub tag to use.",
     )
 
+    parser.add_argument(
```

Comment:
Does this need to be configurable? Prometheus and Grafana are cheap enough that I kinda think we should just turn them on by default.

Reply:
Yes, that is a good idea -- I will experiment with this later because:
- I had a negative experience wirh `jaeger` , which just straight up OOMed on me hosing the machine the test was running on
- due to excessive memory consumption on Mz's part, some of our CI jobs are in the I-am-almost-ooming-on-a-16-gb-machine state. Right now I do not like to disturb the balance as there are a lot of other Buildkite/Release Qualification-related changes that are in flight.

Reply:
Yeah, Jaeger is not nearly so reliable. Sounds good!